target/riscv64.mk

```diff
@@ -16,6 +16,14 @@ CXX := $(CROSS)g++
 OLVL ?= -O2
 CFLAGS += -fomit-frame-pointer -mcmodel=medany
 
+ifeq ($(TARGET_SUBFAMILY), generic)
+  CFLAGS += -march=rv64gch_zicbom_zicboz_zicntr_zihintntl_zihintpause_zihpm_zawrs_zfa_zca_zcd_zba_zbb_zbc_zbs_sstc
+else ifeq ($(TARGET_SUBFAMILY), gr765)
+  CFLAGS += -march=rv64gch_zba_zbb_zbs_zbc_zbkc_zbkb_zbkx_zcb_zkn_zkr_sscofpmf_sstc_ssstateen_smepmp_smaia_ssaia_zicbom_zicond_zfh_zfhmin_zfa
+else ifeq ($(TARGET_SUBFAMILY), grfpga)
+  CFLAGS += -march=rv64gc_zba_zbb_zbs_zbc_zcb_sscofpmf_zicbom_zicond_zfh_zfhmin_zfa
+endif
+
```

Comment:
Perhaps there is a way to pass here a string defined in `_projects/xxx/build.project`?